src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,378 @@
+---
+coding: utf-8
+
+title: Draft 1 - PayID Easy Checkout Protocol
+docname: payid-easy-checkout-protocol
+category: std
+
+pi: [toc, sortrefs, symrefs, comments]
+smart_quotes: off
+
+area: security
+author:
+  -
+    ins: N. Kramer
+    name: Noah Kramer
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: nkramer@ripple.com
+    uri: https://www.ripple.com
+  -
+    ins: D. Fuelling
+    name: David Fuelling
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: fuelling@ripple.com
+    uri: https://www.ripple.com
+  -
+    ins: I. Simpson
+    name: Ian Simpson
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: isimpson@ripple.com
+    uri: https://www.ripple.com
+
+normative:
+    RFC2119:
+    RFC2818:
+    RFC3986:
+    RFC6265:
+    RFC7033:
+    RFC7231:
+    RFC7413:
+    RFC6570:
+    RFC8446:
+    PAYID-PROTOCOL:
+      title: "PayID Protocol"
+      author:
+         ins: A. Malhotra
+         fullname: Aanchal Malhotra
+         ins: D. Schwartz
+         fullname: David Schwartz
+    PAYID-URI:
+      title: "The 'payid' URI Scheme"
+      target: https://tbd.example.com/
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+    PAYID-DISCOVERY:
+      title: "PayID Discovery"
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+
+informative:
+    RFC5988:
+
+--- note_Feedback
+
+This specification is a draft proposal, and is part of the [PayID Protocol](https://payid.org/) initiative. Feedback related to this document should be sent in the form of a Github issue at: https://github.com/payid-org/rfcs/issues.
+ 
+--- abstract
+This specification formalizes how a payment recipient, such as a merchant or a non-profit, can automatically 
+initiate a payment from a payer using only the payer's PayID. 
+
+--- middle
+
+# Terminology
+
+This protocol can be referred to as the `PayID Easy Checkout Protocol`. It uses the following terminology:
+   
+* PayID Easy Checkout Client: A client that queries a PayID Discovery Server using the PayID Discovery Protocol as defined in [PAYID-DISCOVERY][] and that assembles a PayID Easy Checkout URL.
+* PayID Discovery Server: An endpoint that returns a PayID Discovery JRD conforming to the PayID Discovery Protocol as defined in [PAYID-DISCOVERY][].
+* Recipient: An individual or entity receiving a payment (e.g., e-commerce merchant, charity).
+* Payer: An individual or entity originating a payment to a `recipient`.
```

Comment:
I vote we use `Sender` here (and throughout the RFC) instead of `Payer`. 

1) It's kind of a mouthful to say, "PayID Payer".
2) When I say, "The PayID Payer pays the recipient..." I find this awkward. Senders & Receivers go together, and "Payer and Payee" go together. However, "payer and Recipient" don't really go together.

Any strong feelings here?

Reply:
Tomato, tomato.  Guess that saying doesn't really translate well in text.... anyway, no strong feelings, Sender sounds good to me.